### ISSUE TITLE
Unable to recreate a dataset (Bug?)

### ISSUE THREAD

**Issue by HamzahNizami** [jsbroks/coco-annotator#139]
I think I may have stumbled across a bug (or i've done something wrong!).

I created a dataset and imported my images by placing the images locally into the corresponding folder in datasets. After doing so the images were successfully visible online. 

I then proceeded to delete this dataset via the delete function. After doing so the folder remained locally and so I deleted it locally. I then removed/deleted the dataset from the undo area. 

After doing the above steps, I wanted to create the dataset once more with the same name. I successfully did so and continued to place the images into the folder locally. This was also successfully accomplished. However, when refreshing the dataset online it appears that none of my images are available even though the images are in the correct folder. Any idea what I may have done wrong? 

I then created a dataset with a different name and placed the same images into the new dataset folder. These images were then available online, so I am currently under the assumption that I cannot recreate datasets with a name I previously had? Also, would it be possible to have a "rename dataset" function?

I hope I was clear in my explanation! Any help would be appreciated. 
Thanks in advance.

**Comment by jsbroks** [comment]
Have you tried using the re-scan button?

Might be something to do with the dataset ID. Also note, deleting a dataset does not remove the children images from the dataset.

Also create another issue for the feature request of renaming datasets.

**Comment by HamzahNizami** [comment]
Yes, I have tried the scanning button but it doesn't find anything. I have stopped the container and restarted but still no luck.

**Comment by luxedo** [comment]
I had the same issue. Could not reproduce it though, so maybe there's something I'm missing.

The steps that I took that ended up in this error were:
1. I created a dataset and filled with around 60 images
2. Then I imported some annotations to the dataset but with wrong class labels. 
3. I deleted this dataset inside the annotator and created another one.
4. I copied the images from one folder to another, but they didn't appeared.

I solved by deleting the contents of the dataset and copying the images again.
